RepoCleanup/Program.cs

```diff
@@ -38,12 +39,12 @@ static async Task Main(string[] args)
                 repos.AddRange(await GetRepositories(user.Username, null));
             }
 
-            Console.WriteLine($"\r\n Total number of repositories: {repos.Count}");
+            Console.WriteLine($"\r\nTotal number of repositories: {repos.Count}");
```

Comment:
@acn-sbuad Better to use a seperate `Console.WriteLine();` instead of `\r\n` and `\n\n`? Will probably work better cross-platform and the code more readable...